packages/shared/src/components/filters/TagsFilter.tsx

```diff
@@ -23,10 +24,24 @@ import { FilterMenuProps } from './common';
 import MenuIcon from '../icons/Menu';
 import AuthContext from '../../contexts/AuthContext';
 import { useMyFeed } from '../../hooks/useMyFeed';
+import classed from '../../lib/classed';
+import { HTMLElementComponent } from '../utilities';
+
+const TagsContainer = classed('div', 'grid grid-cols-1 gap-4 mx-6');
+
+interface TagsFilterProps extends FilterMenuProps {
+  tagCategoryLayout?: TagCategoryLayout;
+}
+
+const Container: Record<TagCategoryLayout, HTMLElementComponent> = {
+  settings: TagsContainer,
+  default: React.Fragment,
+};
```

Comment:
Is there anyway we can uniform it with the one above?
Looks very similar

Reply:
They have their own way of using their containers. I think it is better to keep them separated from their own files. Also that they are only used in their respective components/files. WDYT?

Reply:
Sure no problem with that 🙏